cypress/e2e/Admin/create/multipleChoices.cy.js

```diff
@@ -24,12 +24,14 @@ import { APP_SETTINGS } from '../../../fixtures/appSettings';
 
 const t = i18n.t;
 
-const { data, id } = APP_SETTINGS.find(
+const { data } = APP_SETTINGS.find(
   ({ name, data }) =>
     name === APP_SETTING_NAMES.QUESTION &&
     data.type === QuestionType.MULTIPLE_CHOICES
 );
 
+const id = data.questionId;
```

Comment:
Unless you need data, you can simplify:
```suggestion
const { data: { questionId: id } } = APP_SETTINGS.find(
  ({ name, data }) =>
    name === APP_SETTING_NAMES.QUESTION &&
    data.type === QuestionType.MULTIPLE_CHOICES
);
```

Reply:
Thanks for the handy simplification! 
I do need data in what comes next though. I'm open if there's another way to simplify further!